net/redis/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2007-2015 OpenWrt.org
```

Comment:
my purpose was to simple reinstate the work of @pdf here 

https://github.com/pdf/openwrt-14.07-x86_64-packages/tree/master/net/redis

since asked from a couple of users of our Arduino internet boards (Yun and Tian). I'll send a private message to this guy if he agrees about the Makefile but I won't see any problem about that...